test/test_transforms_v2.py

```diff
@@ -145,6 +145,7 @@ class TestSmoke:
             (transforms.ConvertBoundingBoxFormat(datapoints.BoundingBoxFormat.CXCYWH), None),
             (transforms.ConvertImageDtype(), None),
             (transforms.GaussianBlur(kernel_size=3), None),
+            (transforms.GaussianBlur(kernel_size=3, sigma=(0.1, 5)), None),
```

Comment:
Why do we need to add it here as well? We already have an entry for `GaussianBlur` above.

Reply:
I'll remove this test as added one to refactored file